sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/models/ier12-integration.model.ts

```diff
@@ -140,4 +141,89 @@ export type IERAward = Pick<
 export const DATE_FORMAT = "YYYYMMDD";
 export const SPACE_FILLER = " ";
 export const NUMBER_FILLER = "0";
+
+/**
+ * Application event code for IER file.
+ */
+export enum ApplicationEventCode {
+  /**
+   * Disbursement was configured as a result of the initial
+   * assessment and waiting for the 21 day prior to study start
+   * date to issue COE request.
```

Comment:
The COE request, in my understanding, is issued when the assessment is generated. The approval is what needs to wait for the 21-day window.
I would say:
```ts
/**
*  Disbursement was configured as a result of the initial
*  assessment and waiting for the 21 days prior to the study
*  start date to be approved.
*/
```

Reply:
Updated, I actually copy pasted these context from the excel sheet